mlrun/api/api/endpoints/operations.py

```diff
@@ -26,14 +28,28 @@ def trigger_migrations(
     background_tasks: fastapi.BackgroundTasks,
     response: fastapi.Response,
 ):
+    # only chief can execute migrations, redirecting request to chief
+    if mlrun.mlconf.httpdb.clusterization.role != "chief":
+        chief_client = mlrun.api.utils.clients.chief.Client()
+        migrations_response = chief_client.trigger_migrations()
+        response.status_code = migrations_response.status_code
+        if migrations_response.status_code == http.HTTPStatus.ACCEPTED.value:
+            response_body = migrations_response.json()
+            return mlrun.api.schemas.BackgroundTask(**response_body)
+        if migrations_response.status_code == http.HTTPStatus.OK.value:
+            return migrations_response.json()
+        else:
+            try:
+                return migrations_response.json()
+            except Exception:
+                return {}
```

Comment:
In general I don't understand the reason for all the conditions here, you should **always** just proxy the response you got from the chief, meaning to return the same status code, body, everything